packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -474,6 +484,41 @@ class NextjsComponent extends Component {
       }
     }
 
+    const regenerationLambdaInput: LambdaInput = {
+      description: inputs.description
+        ? `${inputs.description} (API)`
+        : "Next.js Regeneration Lambda",
+      handler: inputs.handler || "index.handler",
+      code: join(nextConfigPath, REGENERATION_LAMBDA_CODE_DIR),
+      role: inputs.roleArn
+        ? {
+            arn: inputs.roleArn
+          }
+        : {
+            service: ["lambda.amazonaws.com"],
+            policy
+          },
+      memory: readLambdaInputValue(
+        "memory",
+        "regenerationLambda",
+        512
+      ) as number,
+      timeout: readLambdaInputValue(
+        "timeout",
+        "regenerationLambda",
+        10
+      ) as number,
+      runtime: readLambdaInputValue(
+        "runtime",
+        "regenerationLambda",
+        "nodejs12.x"
+      ) as string,
+      name: bucketOutputs.name
```

Comment:
You are using bucket name & region to call the lambda, but only give the name here. Shouldn't you also define region here?

Reply:
Im not sure I follow, im naming the function the same as the bucket so that these can be referenced easily (kinda hacky perhaps?), you can see here where I invoke: https://github.com/serverless-nextjs/serverless-next.js/pull/1028/files/30d0e5113b3fa2f6871aebdb70ae93b450d9c31c#diff-3a1cb611298615fb539fd21d0f14dacf7eb5837a67ca0009dedee1418962cc03R43

Reply:
My point is that you are using the S3 region in the lambda client here: https://github.com/serverless-nextjs/serverless-next.js/pull/1028/files#diff-3a1cb611298615fb539fd21d0f14dacf7eb5837a67ca0009dedee1418962cc03R29

So shouldn't you create the lambda in the same region? Rather than the default us-east-1.

Reply:
Gotcha 👍